build.gradle.kts

```diff
@@ -97,12 +97,14 @@ intellijPlatform {
         description = providers.fileContents(layout.projectDirectory.file("README.md")).asText.map {
             val start = "<!-- Plugin description -->"
             val end = "<!-- Plugin description end -->"
-
             with(it.lines()) {
                 if (!containsAll(listOf(start, end))) {
                     throw GradleException("Plugin description section not found in README.md:\n$start ... $end")
                 }
-                subList(indexOf(start) + 1, indexOf(end)).joinToString("\n").let(::markdownToHTML)
+                subList(indexOf(start) + 1, indexOf(end))
+                    .joinToString("\n")
+                    .replace("](./", "](https://raw.githubusercontent.com/redhat-developer/lsp4ij/main/")
```

Comment:
I think you should have 2 replace:

 * one for images (like you did)
 * one for hyperlink

Reply:
No it applies for both